Config.py

```diff
@@ -50,6 +50,10 @@
 # WARNING: This setting is currently not fully implemented and classed as experimental.
 ENABLE_HTTPS = False
 
+# Which certificate authority to trust when performing requests in HTTPS mode.
+# Defaults to the CA contained within this test suite
+CERT_TRUST_CHAIN = "test_data/BCP00301/ca/intermediate/certs/ca-chain.cert.pem"
```

Comment:
As above, I expected something like :

```python
CERT_TRUST_ROOT_CA = "test_data/BCP00301/ca/certs/ca.cert.pem"
```

Reply:
Thanks, yes that's an error. I'll leave the existing variable in-tact as I suspect it'll be needed for the mocks, but I'll add that one too.